contracts/interfaces/IApostle.sol

```diff
@@ -0,0 +1,7 @@
+pragma solidity ^0.4.24;
+
+contract IApostle {
```

Comment:
Prefer to use IMiner instead, cause not only Apostle can be miners, there are other tokens can be miner.